api/interfaces/Extended.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @interface Extended
+ */
+class Extended {
```

Comment:
I'd split up `Extended` into `ExtendedDiscord` and `ExtendedTS3` as this how it's implemented in the engine and makes the documentation easier to understand.

We can also do this after merging your pull-request though.